src/Extension/CoreExtension.php

```diff
@@ -391,6 +392,7 @@ public function getExpressionParsers(): array
 
             // all literals
             new LiteralExpressionParser(),
+            new OptionalChainExpressionParser(),
```

Comment:
this should be registered near other `get attribute operators`, not in the section labelled as `all literals`